zerver/openapi/zulip.yaml

```diff
@@ -17648,6 +17651,13 @@ components:
             The UNIX timestamp for when the message will be sent
             by the server, in UTC seconds.
           example: 1595479019
+        failed:
+          type: boolean
+          description: |
+            Whether the server has tried to send the scheduled message
+            and it failed to successfully send.
```

Comment:
We can extend this with something about how to use it -- "A client should display these in screens that allow a user to view their scheduled messages and unschedule/copy content from them, both to explain the failure and to allow the user to recovery the message content."

Reply:
Expanded both this and the API changelog entry. See updated screenshots in [this comment](https://github.com/zulip/zulip/pull/25553#issuecomment-1546146873).